### PR TITLE
Update sococo

### PR DESCRIPTION
**Pull request by jansauer**

### CODE REVIEW

**Comment by jansauer** [comment]
PR is based on #11803

File: Casks/sococo.rb [hunk 1]
```diff
@@ -1,11 +1,11 @@
 cask :v1 => 'sococo' do
-  version '3.2.7'
-  sha256 '6772f2c2ef5c7b046612de408b79a71ed0936041fb9286785af69d862f653d1f'
+  version '3.5.9'
```
Comment (vitorgalvao) [inline Casks/sococo.rb]: Should be `3.5.9_13089`.

File: Casks/sococo.rb [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
+  sha256 '5630d397ae3816a965114abf299a594c972773b9de06f21b8b9f9d57d6f1dba6'
 
-  url "http://download.sococo.com/10069/Sococo_#{version.gsub('.','_')}_10069.dmg"
+  url "http://download.sococo.com/3.5.x/Sococo_#{version.gsub('.','_')}_13089.dmg"
```
Comment (vitorgalvao) [inline Casks/sococo.rb]: And this `http://download.sococo.com/#{version_to_f}.x/Sococo_#{version.gsub('.','_')}.dmg`

Comment (jansauer) [inline Casks/sococo.rb]: thx @vitorgalvao

File: Casks/sococo.rb [hunk 3]
```diff
@@ -1,11 +1,11 @@
 cask :v1 => 'sococo' do
-  version '3.2.7'
-  sha256 '6772f2c2ef5c7b046612de408b79a71ed0936041fb9286785af69d862f653d1f'
+  version '3.5.9_13089'
+  sha256 '5630d397ae3816a965114abf299a594c972773b9de06f21b8b9f9d57d6f1dba6'
 
-  url "http://download.sococo.com/10069/Sococo_#{version.gsub('.','_')}_10069.dmg"
+  url "http://download.sococo.com/#{version_to_f}/Sococo_#{version.gsub('.','_')}.dmg"
```
Comment (vitorgalvao) [inline Casks/sococo.rb]: Made a mistake while writing. Should be `#{version.to_f}` (first `_` should be a `.`, instead).

Comment (jansauer) [inline Casks/sococo.rb]: Sorry I should have run `audit` after your changes.

**Pull request merged**